nomad/structs/structs_oss.go

```diff
@@ -0,0 +1,13 @@
+// +build !ent
+
+package structs
+
+import "errors"
+
+func (m *Multiregion) Validate(jobType string, jobDatacenters []string) error {
+	if m != nil {
+		return errors.New("Multiregion jobs are unlicensed.")
+	}
+
+	return nil
+}
```

Comment:
a bit odd new line line here?

Reply:
Yeah the linter picked it up too. Will fix.